### PR TITLE
Convert to string in toString override

### PR DESCRIPTION
**Pull request by Jonathan-Ironman**
The old example did not necessarily return a string.

### CODE REVIEW

**Comment by schalkneethling** [comment]
@Jonathan-Ironman  Because of the following security vulnerability https://snyk.io/blog/malicious-code-found-in-npm-package-event-stream that was revealed yesterday, we are reaching out to everyone that currently has an open PR on interactive examples.

We ask that you please update your fork's master branch, and rebase your current feature branch against the new master. This will update the affected dependency to a version that is no longer affected, and remove the security concern.

We appreciate your assistance in this matter. Should you need any assistance, please feel free to reach out.

File: live-examples/js-examples/object/object-prototype-tostring.html [hunk 1]
```diff
@@ -1,15 +1,19 @@
...
-<code id="static-js">function Dog(name) {
-  this.name = name;
+<code id="static-js">
+class Dog {
+  constructor(name) {
+    this.name = name;
+  }
+  
+  toString() { 
+    return "" + this.name;
```
Comment (wbamberg) [inline live-examples/js-examples/object/object-prototype-tostring.html]: We use single quotes for strings.

Comment (Jonathan-Ironman) [reply live-examples/js-examples/object/object-prototype-tostring.html]: Fixed

File: live-examples/js-examples/object/object-prototype-tostring.html [hunk 2]
```diff
@@ -1,15 +1,19 @@
...
+    return "" + this.name; 
+  }
 }
 
-dog1 = new Dog('Gabby');
-
-Dog.prototype.toString = function dogToString() {
-  return this.name;
-}
+const dog1 = new Dog("Gabby");
```
[the remark this answers is not in the document]
Comment (Jonathan-Ironman) [reply live-examples/js-examples/object/object-prototype-tostring.html]: Fixed

**Pull request closed without merging**